servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsClient.java

```diff
@@ -192,6 +193,12 @@ final class DefaultDnsClient implements DnsClient {
         resolver = builder.build();
     }
 
+    @Override
+    public String toString() {
+        return getClass().getSimpleName() + // FIXME: change to the name/id when builder requires it
```

Comment:
This will be "id" we discussed with you when you will add ServiceLoader provider